helmfile/charts/starboard/ciskubebench-exporter/values.yaml

```diff
@@ -0,0 +1,11 @@
+ciskubebenchExporter:
+  schedule: "* * * * *"
+resources:
+  limits:
+    cpu: 100m
+    memory: 128Mi
+  requests:
+    cpu: 50m
+    memory: 64Mi
+tolerations: []
+affinity: {}
```

Comment:
You seem to be missing the cronjob resources here.

Reply:
Both are using the same resources. Do you want me to split it?